styled/styled.go

```diff
@@ -0,0 +1,101 @@
+package styled
+
+import (
+	"fmt"
+)
+
+type Style struct {
+	Foreground string
+	Background string
+	Bold       bool
+	Dim        bool
+	Italic     bool
+	Underlined bool
+	Blink      bool
+	Inverse    bool
+}
+
+func EmptyStyle() Style {
+	return Style{
+		Foreground: "default",
+		Background: "default",
+	}
+}
+
+func (s *Style) ImportFromOptions(options map[string]interface{}) error {
+	assignColor := func(key string, assign func(string)) error {
```

Comment:
The second argument can be a `*string`.

Reply:
I knew there has to be a non awkward way to do this. Thx!